lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2044,6 +2084,7 @@ mkApiTransaction ti txid mfee ins outs ws (meta, timestamp) txMeta setTimeRefere
         , inputs = [ApiTxInput (fmap toAddressAmount o) (ApiT i) | (i, o) <- ins]
         , outputs = toAddressAmount <$> outs
         , withdrawals = mkApiWithdrawal @n <$> Map.toList ws
+        , forge = mempty  -- fixme: ADP-604
```

Comment:
Question: Should this be `mint`? And do we need to support this particular field as part of ADP-604?

```suggestion
        , mint = mempty  -- fixme: ADP-604
```

Reply:
Ok thanks - i'll use "mint" to match the terminology from Cardano.Api.
The API field is a placeholder to help Daedalus team with integration.

The correct place to review these names is in #2446 - because ApiTransaction just mirrors what's in `swagger.yaml`.